packages/slate-editor/src/modules/editor/types.ts

```diff
@@ -99,7 +99,6 @@ export interface EditorProps {
                   embed?: boolean;
                   link?: boolean;
```

Comment:
AFAIK, `link` shouldn't be used too. Please check.

Reply:
Looks like all of these props are unused